portfolio-page.html

```diff
@@ -0,0 +1,166 @@
+				<figure class="project-figure">
+					<img class="project-image" src="images/terminal-wizard.jpeg" 
+						alt="A picture for project-4">
+					<figcaption class="project-caption">Project-4</figcaption>
+				</figure>
+				<p class="project-info">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus est mauris, aliquam in felis a, ullamcorper ornare est. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Aliquam erat volutpat. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Cras vulputate ligula neque, nec faucibus lacus auctor nec. Sed accumsan fringilla neque, vel mollis ligula interdum quis. Sed turpis mi, volutpat at pretium id, porttitor eget nunc. Fusce convallis dolor bibendum finibus finibus. Pellentesque rutrum lobortis ex eu fermentum.</p>
+			</article>
+		</section> <!-- .portfolio-section -->
+
+		<section class="cheat-sheets-section" id="cheat-sheets">
+			<h2 class="section-title">Cheat Sheets</h2>
+			<article class="cheat-sheet">
+				<h3 class="cheat-sheet-title">HTML</h3>
+				<figure class="cheat-sheet-figure">
+					<iframe src="cheat-sheets/HTML-cheatsheet.pdf"></iframe>
+					<figcaption class="cheat-sheet-caption">
+						<a class="cheat-sheet-link" target="_blank" href="https://www.notion.so/HTML-Cheatsheet-a794169a8960478e891b3e4ee8b1814c">Link</a>
+					</figcaption>
+				</figure>
+			</article>
+			<article class="cheat-sheet">
+				<h3 class="cheat-sheet-title">CSS</h3>
+				<figure class="cheat-sheet-figure">
+					<iframe src="cheat-sheets/CSS-cheatsheet.pdf"></iframe>
+					<figcaption class="cheat-sheet-caption">
+						<a class="cheat-sheet-link" target="_blank" href="https://www.notion.so/CSS-Cheatsheet-11d3d3e7771d4ac49c4f5606dee840a3">Link</a>
+					</figcaption>
+				</figure>
+			</article>
+			<article class="cheat-sheet">
+				<h3 class="cheat-sheet-title">Git & Github</h3>
+				<figure class="cheat-sheet-figure">
+					<iframe src="cheat-sheets/git-github.pdf"></iframe>
+					<figcaption class="cheat-sheet-caption">
+						<a class="cheat-sheet-link" target="_blank" href="https://www.notion.so/Git-GitHub-c693bec482184234b6c955d7ce54dff1">Link</a>
+					</figcaption>
+				</figure>
+			</article>
+			<article class="cheat-sheet">
+				<h3 class="cheat-sheet-title">Command Line</h3>
+				<figure class="cheat-sheet-figure">
+					<iframe src="cheat-sheets/command-line.pdf"></iframe>
+					<figcaption class="cheat-sheet-caption">
+						<a class="cheat-sheet-link" target="_blank" href="https://www.notion.so/Command-Line-Matrix-Time-3079064d9f334bf79de54595383fb271">Link</a>
+					</figcaption>
+				</figure>
+			</article>
+			<article class="cheat-sheet">
+				<h3 class="cheat-sheet-title">Tech Concepts</h3>
+				<figure class="cheat-sheet-figure">
+					<iframe src="cheat-sheets/tech-concepts.pdf"></iframe>
+					<figcaption class="cheat-sheet-caption">
```

Comment:
lots of classes that are not used in the css - all the class names appear to be semantic which is very good, however we only use classes when they are needed, so I'd suggest you go through all the html and remove all classes that are not used for anything

Reply:
oh i think some of them are used - anyway yeah if you can remove any classes that are not used